mautic/mautic
# Send Webhook Contact Updated Event just on updated contact

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Update contact webhook and send payload on each save entity, even without changes. The PR fixed it.




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create webhook for update contact event and use https://webhook.site/ generate webhook url
3. Go to contact profile and open and save contact without changes
4. You see webhook payload on  https://webhook.site/
5. After PR should send changes just on real contact data changes
